test/sslapitest.c

```diff
@@ -37,6 +38,10 @@
 #include "internal/ktls.h"
 #include "../ssl/ssl_local.h"
 
+/* Defined in filterprov.c */
+OSSL_provider_init_fn filter_provider_init;
+int filter_provider_set_filter(int operation, const char *name);
```

Comment:
This almost makes me wonder if there should be a setparams for providers.

Reply:
Perhaps...but not for this PR!!